lib/membrane_h264_plugin/parser.ex

```diff
@@ -64,6 +64,15 @@ defmodule Membrane.H264.Parser do
                 Picture Parameter Set NAL unit binary payload - if absent in the stream, should
                 be provided via this option.
                 """
+              ],
+              framerate: [
+                spec: {pos_integer(), pos_integer()},
+                default: {30, 1},
+                description: """
```

Comment:
I'm not sure about the default value of this option. 

IMO if framerate is uknown / inconstant parser should not sent the framerate in caps (this requires changing the H264 format) or pass `{0, 1}`

Reply:
Maybe returning `{0, 0}` would be okay? It would refere to `undefined` in math :thinking:

Reply:
I have set the default to `nil`